posts/how-to-add-open-source-software-projects-to-sourcegraph-cloud.md

```diff
@@ -5,7 +5,7 @@ authorSlug: lisa-tagliaferri
 authorDisplayName: Lisa Tagliaferri
 publicationDate: May 11, 2021
 description: Learn how to add public, open source repositories to Sourcegraph.
-image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-header-2.png
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-13.png
```

Comment:
I decided to update the banners on the homepage to make it more visually engaging